### PR TITLE
added plugin: shufflecollection

### PR DESCRIPTION
**Pull request by rain0r**
Added a plugin which shuffles through the whole collection. It add a song from the collection dynamically to the current playlist. Code style might be ugly but it's working.

### CODE REVIEW

File: plugins/shufflecollection/__init__.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    settings,
+    trax
+)
+from xl.player.adapters import (
+    PlaybackAdapter,
+    QueueAdapter
+)
+from xlgui import main
+from xlgui.widgets import menu
+import glib, gtk, logging, os, random, time, thread
```
Comment (virtuald) [inline plugins/shufflecollection/__init__.py]: Have you tested against master? We're using GTK 3.x and PyGObject 3.x now.

**Comment by rain0r** [comment]
Yeah, unfortunately it doesn't work with 4.0.0-dev but with 3.4.5 though.

**Comment by virtuald** [comment]
We won't be able to include this in master if it doesn't work in master.

**Comment by virtuald** [comment]
Closing this PR. If you port to Exaile 4.0, feel free to resubmit it!

**Pull request closed without merging**